labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -50,7 +61,14 @@ object CollectionExercise02 {
    * using a functional approach.
    */
   def groupAdultsPerAgeGroup(persons: Seq[Person]): Map[Int, Seq[Person]] = {
-    error("fix me")
+    var cutOff: Int = 10
+    var ageGroups = scala.collection.mutable.Map[Int, Seq[Person]]()
+    
+    while(cutOff < 100) {
+      ageGroups.put(cutOff, persons.filter(_.age < cutOff))
+      cutOff += 10
+    }
```

Comment:
Good use of `filter`!

You probably could have done even better using `groupBy` instead of cycling through all the cutOffs